app/assets/src/components/visualizations/table/InfiniteTable.jsx

```diff
@@ -72,7 +73,12 @@ class InfiniteTable extends React.Component {
     // Guarantees that we have at least one child div in the cell
     return (
       <div className={cs.cellContent}>
-        {cellData == null ? "" : String(cellData)}
+        {/* Use .name if val is an object (e.g. location object) */}
```

Comment:
Why do you need this?

Reply:
this one was because the location object is getting passed down altogether (like in other places) so it would show up as [object Object]. I could make a locationRenderer instead here?